test/smoke.py

```
@@ -165,12 +165,13 @@ def main():
 
         print("Connection header matching")
 
-        def conn(extra):
+        def conn(extra, version=b"1.1"):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `conn`

(ANN202)